.editorconfig

```diff
@@ -271,8 +277,8 @@ dotnet_style_null_propagation = true:error
 dotnet_style_prefer_is_null_check_over_reference_equality_method = true:suggestion
 dotnet_style_prefer_auto_properties = true
 dotnet_style_object_initializer = true:suggestion
-dotnet_style_collection_initializer = true
+dotnet_style_collection_initializer = true:suggestion
```

Comment:
It looks like I duplicated things after accepting Visual Studio's suggestions in a previous commit.  I'll address this in another commit later.